models/iaf_psc_alpha.h

```diff
@@ -40,71 +40,98 @@ namespace nest
 Short description
 +++++++++++++++++
 
-Leaky integrate-and-fire neuron model
+Leaky integrate-and-fire model with alpha-shaped input currents
 
 Description
 +++++++++++
 
-``iaf_psc_alpha`` is an implementation of a leaky integrate-and-fire model
-with alpha-function shaped synaptic currents. Thus, synaptic currents
-and the resulting postsynaptic potentials have a finite rise time.
+``iaf_psc_alpha`` is a leaky integrate-and-fire neuron model with
 
-The threshold crossing is followed by an absolute refractory period
-during which the membrane potential is clamped to the resting potential.
+* a hard threshold,
+* a fixed refractory period,
+* no adaptation mechanisms,
+* :math:`\alpha`-shaped synaptic input currents.
```

Comment:
Please consider to make the descriptions in the list more expressive. Right now they are very short and might be intimidating to new users.

Reply:
@jougs I was aiming for a concise overview here, assuming that our users are computational neuroscientists. Especially for users looking for if a model fits their needs, such a brief bullet list with quite standardized terms seems useful to me.

Reply:
In my view, our users are either computational neuroscientists already *or* people who want to become one of those.

I like the brevity for the former and agree that standardized terms are a *good thing*<sup>TM</sup>, but I'm also afraid that for the latter, terms like *hard threshold* (compared to what?), *α-shaped synaptic input currents* (how does this shape look like? what other shapes could there be?) and *no adaptation mechanisms* (what is not adapted? and what could be in other models?) might raise a lot of questions and could be too intimidating.

I suggest to add at least some links to [the glossary](https://nest-simulator.readthedocs.io/en/v3.5/ref_material/glossary.html) or suitable Scholarpedia/Wikipedia articles here.

Reply:
I also agree with @jougs that we need to provide some more context for users less familiar with these terms.
Can we expand on this bullet list in a subsequent paragraph?  
We could add a link to the glossary, but do users need a standard definition of these terms or do they need to know how these terms are used within this model? In the latter case, I think the glossary is insufficient.

Reply:
@jessica-mitchell: do you think a kind of "reverse glossary" would be possible as a Sphinx extension? Something along the lines of what you did in #2859, but for listing all the documentation pages where a term is used next to the entry in the glossary (in some kind of hidden sliding block thingy). That could possibly solve the problem you outline in an elegant and semi-automatic way.

Reply:
@jougs yes, that should be possible- maybe for now we can leave the documentation as @heplesser suggests and I can look into developing something like this

Reply:
@jessica-mitchell: can you please create an issue so we don't forget about this?